website/src/pages/api/update_task.ts

```diff
@@ -49,28 +53,13 @@ const handler = withoutRole("banned", async (req, res, token) => {
   });
 
   const user = await getBackendUserCore(token.sub);
-  let newTask;
   try {
-    newTask = await oasstApiClient.interactTask(update_type, taskId, frontendId, interaction.id, content, user!, lang);
+    await oasstApiClient.interactTask(update_type, taskId, frontendId, interaction.id, content, user!, lang);
+    return res.status(204).send("");
   } catch (err: unknown) {
     console.error(JSON.stringify(err));
     return res.status(500).json(err);
   }
-
-  // Stores the new task with our database.
-  const newRegisteredTask = await prisma.registeredTask.create({
-    data: {
-      task: newTask,
-      user: {
-        connect: {
-          id: token.sub,
-        },
-      },
-    },
-  });
-
-  // Send the next task in the sequence to the client.
-  res.status(200).json(newRegisteredTask);
```

Comment:
The backend doesn't return a new task here (this looks like a legacy behavior) and the client doesn't consume this response so I removed it

Reply:
it would be really great to know what was the intention behind this?
we are we tracking the registered tasks in the db? did we do (or wanted to do) any this with the information?

Reply:
Just remove legacy/unnecessary code. Here is what `newTask` variable contain
```
{
  "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
  "type": "task_done"
}
```
We are storing unwanted records. I'm not sure why we store newTask in our DB. Maybe it is a legacy behavior. And in the client, it doesn't consume the response of this endpoint. I will use `/api/new_task` to receive a new task.